tests/wpt/web-platform-tests/url/urlsearchparams-constructor.any.js

```diff
@@ -147,6 +147,10 @@ test(function() {
     assert_equals(params.get('a\uD83D\uDCA9b'), 'c');
 }, 'Parse %f0%9f%92%a9');  // Unicode Character 'PILE OF POO' (U+1F4A9)
 
+test(() => {
```

Comment:
I think a more appropriate place for this test might be tests/wpt/web-platform-tests/WebIDL/ecmascript-binding/sequence-conversion.html or one of the other files in that directory.